src/Tests/dotnet-new.Tests/Approvals/DotnetNewListTests.CanShowMessageInCaseShortNameConflict.verified.txt

```diff
@@ -0,0 +1,8 @@
+﻿These templates matched your input: 
+
+Template Name                  Short Name                                   Language  Tags      
+-----------------------------  -------------------------------------------  --------  ----------
+TemplateWithConflictShortName  list,search,install,uninstall,update,create            Test Asset
+
+To instantiate the template with short name 'list', run:
+   dotnet new create list
```

Comment:
@baronfel please check the wording here.

Reply:
This LGTM 🚢